Technical-Steering-Committee/principles.md

```diff
@@ -0,0 +1,43 @@
+# Principles
+
+Zowe is a project that aims to bring modern interfaces - API, command line, and visual – to z/OS via an open source community. In case of disagreement within the Zowe community, the principles outlined here need to be taken into account when mediating.
+
+## Zowe simplifies the life of Modern Mainframers
+**Principle:** Modern and effective interfaces
+
+We will provide the foundations and tools for all the vendors existing within the ecosystem to easily interface to and develop more interfaces to z/OS. The users should be able to leverage the provided tools and interfaces to get more value out of their mainframe environments. Examples domains for such tools and interfaces include web, mobile, APIs (C interfaces to HLASM, other programming interfaces, REST APIs, ...) and modern IDEs.
+
+## Zowe does not compromise on z/OS security standards
+**Principle:** Security is paramount
+
+We commit to secure development practices, source code scans and vulnerability testing. We will never introduce a feature or change that circumvents or allows circumvention of z/OS security infrastructure. If vulnerabilities are ever found they will be flagged with fix or workaround provided.
```

Comment:
May I propose a little rewording of this sentence:

> We will never **intentionally** introduce a feature or a change that would circumvent or would allow circumventing the z/OS security infrastructure. If vulnerabilities would be ever found they would be flagged with a fix or a workaround provided.

Reply:
Sure, that sounds good to me.